custom-dashboards/actions.js

```diff
@@ -0,0 +1,86 @@
+const core = require('@actions/core');
+const execa = require('execa');
+const glob = require('@actions/glob');
+const fs = require('fs/promises');
+const _fs = require('fs');
+
+async function run() {
```

Comment:
We can delete this file, but please explain once how it works in the KT, seems some real good work :)